### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,14 @@
-/* styles go here */
+.fieldset {
+  box-sizing: border-box;
```
Comment (OleziO) [inline src/style.css]: Move this to `*` selector

File: src/style.css [hunk 2]
```diff
@@ -1 +1,14 @@
...
+  margin-bottom: 20px;
+}
+
+.form-field {
+  box-sizing: border-box;
```
Comment (OleziO) [inline src/style.css]: Remove this line after adding to `*` selector.

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,164 @@
...
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="#" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+
```
Comment (OleziO) [inline src/index.html]: Remove unnecessary empty line.

Comment (OleziO) [reply src/index.html]: Check other similar cases.

File: src/index.html [hunk 4]
```diff
@@ -11,7 +11,164 @@
...
+            minlength="5"
+            maxlength="35"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
```
Comment (OleziO) [inline src/index.html]: Remove unnecessary empty line.

**Review by OleziO — CHANGES_REQUESTED** [review]

**Review by sTorba24 — APPROVED** [review]
Good work!

**Pull request closed without merging**